recipes/foamalgo/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "foamalgo" %}
+{% set version = "0.0.4" %}
+{% set sha256 = "79c3dd9f54ea115b7d5528bd4556db70596f5190f39811af0b79346ed549207d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/zhujun98/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win or py==27]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - pip
+    - h5py >=2.10.0
+    - numpy >=1.16.1
+    - scipy >=1.2.1
+    - xsimd >=7.4.8
+    - tbb >=2019.9,<2021.0
+    - tbb-devel
```

Comment:
it seems you disabled xsimd and tbb, should you remove these dependencies then?

Reply:
C++ part is a head-only library so we do not need xsimd and tbb during installation. But in Python, it is included by default.

Reply:
It is a bit different from many other libraries that the Python binding links to the installed C++ libraries.